tests/format/markdown/splitCjkText/__snapshots__/format.test.js.snap

```diff
@@ -22,25 +22,20 @@ nasalカ゚キ゚ク゚ケ゚コ゚カ゚キ゚ク゚ケ゚コ゚カ゚キ゚ク
 かつてはワ行のワ、ヰ、ヱ、ヲに濁点を付して [v] 音を表現すること（ワ゛、ヰ゛、ヱ゛、ヲ゛）も行われたが、一般的にはならなかった。
 
 =====================================output=====================================
-這是一段很長很長很長很長很長很長很長很長很長很長很長很長很長很長很長很長很長很長
-很長的段落
+這是一段很長很長很長很長很長很長很長很長很長很長很長很長很長很長很長很長很長很長很長的段落
 
-全　　形　空白全　　形　空白全　　形　空白全　　形　空白全　　形　空白
-全　　形　空白全　　形　空白
+全　　形　空白全　　形　空白全　　形　空白全　　形　空白全　　形　空白全　　形　空白全　　形　空白
 
-空白全形空白全形空白全形空白　空白全形空白全形空白全形空白　空白全形空白全形空白
-全形空白　空白全形空白全形空白全形空白
+空白全形空白全形空白全形空白　空白全形空白全形空白全形空白　空白全形空白全形空白全形空白　空白全形空白全形空白全形空白
 
 何でも薄暗いじめじめした所でニャーニャー泣いていた事だけは記憶している。
 
 カ゚キ゚ク゚ケ゚コ゚でガギグゴ
 
-nasalカ゚キ゚ク゚ケ゚コ゚カ゚キ゚ク゚ケ゚コ゚カ゚キ゚ク゚ケ゚コ゚カ゚キ゚ク゚
-ケ゚コ゚カ゚キ゚ク゚ケ゚コ゚カ゚キ゚ク゚ケ゚コ゚カ゚キ゚ク゚ケ゚コ゚カ゚キ゚ク゚
-ケ゚コ゚カ゚キ゚ク゚ケ゚コ゚カ゚キ゚ク゚ケ゚コ゚カ゚キ゚ク゚ケ゚コ゚
+nasalカ゚キ゚ク゚ケ゚コ゚カ゚キ゚ク゚ケ゚コ゚カ゚キ゚ク゚ケ゚コ゚カ゚キ゚ク゚ケ゚コ゚カ゚キ゚ク゚ケ゚コ゚カ゚キ゚ク゚ケ゚コ゚カ゚キ゚ク゚ケ゚コ゚カ゚キ゚ク゚ケ゚コ゚カ゚キ゚ク゚ケ゚コ゚カ゚キ゚ク゚ケ゚コ゚カ゚キ゚ク゚ケ゚コ゚
 
-かつてはワ行のワ、ヰ、ヱ、ヲに濁点を付して [v] 音を表現すること（ワ゛、ヰ゛、ヱ
-゛、ヲ゛）も行われたが、一般的にはならなかった。
+かつてはワ行のワ、ヰ、ヱ、ヲに濁点を付して [v]
+音を表現すること（ワ゛、ヰ゛、ヱ゛、ヲ゛）も行われたが、一般的にはならなかった。
```

Comment:
When `prev` or `next` is undefined, `" "` and `"\n"` have been interchangeable.

https://prettier.io/playground/#N4Igxg9gdgLgprEAuEhT00DTmg8qIKJQOYA2AlgM4AWGO+x5AOlJfboaWfQFTuWdPWuNROzGmU4CABsNbj6lKXSjjKMqPLJKskvuTnb1ykABoQEAA4wi0EslABDAE72IAdwAKDhNZS2Cz2wE9rYwAje1swAGs4GABlWwBbOAAZIig4ZAAzHxI4ELDI6JjTcNS8ZBh7AFdckBz4onKqmrgAD1M4eyJE2B8AFQ6oByI4LyyCHOMSUoI4AEVKiHhM7JqAKxIWmOm5haWkMYmQAEdduFcnUy8QWxIAWjS4ABMnoxAK2yJifABhCHj42zIEAjACsrym1DgAEEYBUiMFKvBXB0UmlluMamQYPECAB1MhEeAkYpgOAxTyEogAN0J-iBYBIQRAVOqAEkoM9YDEwJ1zFCOTEYP4ZujDqYnDlcWFTECfH5Aq9Ujl7DBzrY8ADRTVivZlUCAfYIo8XFBXuLUjBcURHjAyMgACwABmM9jgJyIrrVGsB+xWxhgtmCVptdqQACZjJUcr1A6M-cD4sEns9HklbPhKuq4AAxCD2AGw0qyxEQEAAXzLQA